templates/plans/index.hamlet

```diff
@@ -0,0 +1,13 @@
+$forall plan <- plans
+  <div .plan .col-md-4>
+    <div .panel.panel-default>
+      <div .panel-heading>
+        <h3 .panel-title>#{show plan}
+      <div .panel-body>
+        ^{planFeatures plan}
+
+<h2>Why Pay?
+
+<p>Carnival will never contain ads
+<p>Carnival will never track your behavior
```

Comment:
Yeah, I suppose this has to be qualified but I don't know how to say it. The big value-add over Disqus is that we don't collect / monetize on user behavior, so I want to say that -- but clearly we're going to be tracking usage statistics for our own business metrics, so where's the line?